packages/template-retail-react-app/app/ssr.js

```diff
@@ -34,11 +34,11 @@ const options = {
 const runtime = getRuntime()
 
 const {handler} = runtime.createHandler(options, (app) => {
```

Comment:
i guess the behavior of enabling default headers seems more natural to me if it’s a configuration value…
```js
const options = {
   ...,
   enableDefaultSecurityHeaders: true
}
```